src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -13,33 +13,33 @@ cover_image_alt: ugrc social card
 
-When you view aerial imagery through Discover, you may think you are looking at a one really big image of Utah. But, this giant photo of Utah is actually composed of hundreds of flight paths captured by airborne cameras, all mosaicked together into a series of 'tiles' (just like in a real mosaic floor or ceiling, it takes many colorful tiles to make up the whole artwork). These tiles are pre-cached, which allows for seamless transitions as you pan or zoom between them. They render so quickly, that many users don't even realize there is more than one tile to begin with.
+When you view services through Discover, you may think you are looking at a one really big image of Utah. But, this giant photo of Utah is actually composed of hundreds of flight paths captured by airborne cameras, all mosaicked together into a series of 'tiles' (just like in a real mosaic floor or ceiling, it takes many colorful tiles to make up the whole artwork). These tiles are pre-cached, which allows for seamless transitions as you pan or zoom between them. They render so quickly, that many users don't even realize there is more than one tile to begin with.
 
 However, like ragtag siblings stuck in their “get-along” shirt, some of these tiles didn't quite play nice at first. Some of them were so-called 'sticky tiles': when the user zoomed in or out, the other nearby tiles would zoom in or out as well, but sticky tiles would be stuck at the previous zoom level as though glued in place. Other problem children included 'delinquent tiles', which were tiles that didn't show up in the service at all. Apparently, those tiles had somewhere better to be. And many of the other perfectly innocent tiles were hampered by slow rendering speeds.
 
-Luckily, through our team's many long hours of dedicated troubleshooting, these issues were resolved and new solutions were implemented. Nowadays, all the tiles sit amicably at the dinner table and do as they're told, which is to provide a seamless user experience for those who rely on aerial imagery to do their jobs. The Discover service available today is much more robust than the old ArcGIS Server, is cloud-based, provides auto-scaling, and is built to last.
+Luckily, through our team's many long hours of dedicated troubleshooting, these issues were resolved and new solutions were implemented. Nowadays, all the tiles sit amicably at the dinner table and do as they're told, which is to provide a seamless user experience for those who rely on aerial imagery and basemaps to do their jobs. The Discover services available today are much more robust than those from the old ArcGIS Image Server. It is cloud-based, provides auto-scaling, and is built to last. AND WE NEED MORE FROM APPGEO HERE!
```

Comment:
what kind of information do you want from app geo? I can give you more cloud service jargon if you want.

Reply:
I tried for a couple weeks for them to get me a technical sentence from Calvin and all I got was `It's built using Node.js and a Postgres database. ` which works for me